FreeRTOS/FreeRTOS
# Remove 64 Bit Visual Studio Configurations

Description
-----------
The Visual Studio Projects have a configuration option for "Release" and for "x64". These are not actively worked on or maintained. These configurations were generated when creating the project structure with the static library visual projects. Additionally, they all have the `WIN32` flag set in the pre-processor, leading to potential issues when run as a 64 bit application. As such they should be removed from the repository, so as not to mislead people who may attempt to run them. 

Test Steps
-----------
Removed the configuration options. Then built the demos locally to ensure the projects built.
Used GitHub CI-CD as well as internal CI-CD to ensure the demos run correctly.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
